packages/bpk-component-progress/src/BpkProgress.module.scss

```diff
@@ -18,15 +18,17 @@
 
 @import '~bpk-mixins/index';
 
+$bpk-spacing-v2: true;
+
 .bpk-progress {
   position: relative;
-  height: $bpk-spacing-sm;
-  border-radius: $bpk-border-radius-sm;
+  height: bpk-spacing-base();
+  border-radius: $bpk-border-radius-pill;
   background-color: $bpk-color-sky-gray-tint-06;
   overflow: hidden;
 
   &--small {
-    height: $bpk-spacing-xs;
+    height: bpk-spacing-md();
     border-radius: $bpk-border-radius-xs;
```

Comment:
We could actually leave this out so that it uses the bigger radius for both sizes?